### ISSUE TITLE
progress from discord

### ISSUE THREAD

**Issue by afiaka87** [lucidrains/paracusia-insilico#1]
Hey!

Two users in the dalle-pytorch discord have been working on this a bit in a colab in case you needed a kickstart. 

https://colab.research.google.com/drive/14CWRq-sKSVAgaSqcD77mboRcEqMYogs4

**Comment by voodoohop** [comment]
Hi! thanks for making this connection. It's @nielsrolf and me who were doing some experiments yesterday on that colab notebook.

We tried going directly from image to sound but so far only noise with hints of structure emerging sometimes.

**Comment by voodoohop** [comment]
This is an image
![heartbeat](https://user-images.githubusercontent.com/5099901/124382206-85b81080-dcc6-11eb-8b87-215d14737910.png)

And the resulting sound after a while:
https://soundcloud.com/thomash_voodoohop/heart-beat-image-to-sound/s-3erpo2KfzMa

**Comment by afiaka87** [comment]
That's quite interesting; certainly a rythmic thumping bass at a few moments but it's all rather subjective. 

Maybe we could look at spectrograms of the output as well? Compared, say, to a baseline of a spectrogram of an actual heartbeat sound?

**Comment by voodoohop** [comment]
Yes I was going to try a simple text prompt first... Will get around to it a bit later today. People were getting some slightly better results by directly optimizing the waveform instead of a SIREN representation of it.

It could also make sense to try optimizing something in the fourier space but I always reach my math limits quickly.

**Comment by afiaka87** [comment]
Yeah this would be my first shot at anything to do with audio and ML. 

In similar vein to the CLIP+VQGAN and Big Sleep approach; is there a similar analog for audio? A pretrained audio GAN you could move through using the AudioCLIP similarity as the loss?

This would be a good way to keep your audio in the realm of realistic audio outputs hopefully with less noise.

**Comment by voodoohop** [comment]
Yes, I'm actually working on generative audio synthesis in a startup. We are training models that are specialized on reproducing the sounds of certain instruments. Not generic audio generation.

But there is a lot of research. Most of the models are quite heavy that's why we were hoping of getting some rough results nusing SIREN before moving on to something more specialized.

**Comment by nielsrolf** [comment]
I got [this](https://soundcloud.com/nielsrolf/audioclip-bird/s-GAIbm6Wp22J) kind of cool output from [this](https://www.sciencemag.org/sites/default/files/styles/article_main_image_-_1280w__no_aspect_/public/bird_16x9_3.jpg?itok=SR849n1r) input.

**Comment by afiaka87** [comment]
@nielsrolf @voodoohop Looks like @lucidrains has just updated both the deep-daze repositories and the SIREN-pytorch repository ;)

**Comment by voodoohop** [comment]
@afiaka87 I couldn't see any updates yet

**Comment by nielsrolf** [comment]
@afiaka87 We also created a repo for our attempt, if you'd like to follow or contribute: https://github.com/pollinations/CLIPTranslate
